sklearn/decomposition/kernel_pca.py

```diff
@@ -137,12 +155,22 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         and Klaus-Robert Mueller. 1999. Kernel principal
         component analysis. In Advances in kernel methods,
         MIT Press, Cambridge, MA, USA 327-352.
+
+    For eigen_solver == 'arpack', refer to `scipy.sparse.linalg.svds`.
+
+    For eigen_solver == 'randomized', see:
+        `Finding structure with randomness: Stochastic algorithms
```

Comment:
I don't think this use of backticks is appropriate in Restructured Text

Reply:
Sorry, I am a newbie in rst / sphinx - I use markdows+mkdocs in my own open source projects. Here I simply copied what I saw in PCA. Should I change it ? If so we should change PCA doc too.